include/psa/crypto_driver_contexts.h

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Declaration of context structures for use with the PSA driver wrapper
+ *  interface.
+ *
+ *  Warning: This file will be auto-generated in the future.
+ */
```

Comment:
Please state that this header and its content are not part of the Mbed TLS API and applications must not depend on it.